sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/pubsub/PubsubJsonTableProvider.java

```diff
@@ -54,125 +47,29 @@ public String getTableType() {
   }
```

Comment:
I am not sure if we expect the table type here to be the same as `PubsubSchemaCapableIOProvider.identifier()`. WDYT Brian?

Reply:
Yes my intention was that they should be the same. I'm not that concerned about it being hard-coded for now. We're planning to replace this class with a general-purpose "wrapper" that can adapt any `SchemaCapableIOProvider` to the `TableProvider` interface, and I figured `getTableType` would just call `SchemaCapableIOProvider#identifier`.

That work is tracked in BEAM-10337

Reply:
Sounds good!